_config/dataservices.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Just a reminder that we should figure out our config strategy here. What goes in config file committed to repo versus environment (secrets)? How do we handle different configuration for different environments in the config file committed to repo?